api/src/graphql/resolvers/index.js

```diff
@@ -106,45 +108,53 @@ const resolvers = {
     },
   },
   Mutation: {
-    createNeed: combineResolvers(
```

Comment:
Not a fan of these formatting changes. They serve no purpose except preference and make committs/merges more complicated for no reason. Also, I think it's better to keep the existing formatting to keep things consistent and easier to read when we add more authorization steps. My suggestion is to go back to the other formatting for all resolvers. What do you think?

Reply:
I agree, but hadn't noticed the extent of it before you pointed it out. 
This stems from a commit during the hackathon weekend, and probably comes from someones editor enthusiastically trying to 'fix' formatting. It's back to normal now. Thanks for pointing it out! :)